sale_report_hide_price/models/bill.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+# © 2016 Serpent Consulting Services Pvt. Ltd. <sebastien.beau@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api
+from openerp import fields, models
+
+class BillOrder(models.Model):
```

Comment:
You should make new module if its invoice report hiding it.
The name of the module should match the purpose.
We would appreciate a new module if you make.

Reply:
Sure, but, if our customers wants to hide price on sale orders, they want to hide this prices on invoice too. For them, it's obvious to hide price on invoice too.
Maybe the module name can be changed too ?